slick/src/main/scala/akka/stream/alpakka/slick/javadsl/Slick.scala

```diff
@@ -88,6 +109,30 @@ object Slick {
       .map(Int.box(_))
```

Comment:
```suggestion
      .map(Int.box)
```

This is giving warning in IntelliJ, not essential though

Reply:
✅ Done